eg-next/README.md

```diff
@@ -76,3 +76,7 @@
 
 [![A screenshot of the embedded-graphics version next example called text-transparent](../doc/assets/next/text-transparent.png)](examples/text-transparent.rs)
 
+## [text-textbox](examples/text-textbox.rs)
+
+[![A screenshot of the embedded-graphics/embedded-text version next example called text-textbox](../doc/assets/next/text-textbox.png)](examples/text-textbox.rs)
```

Comment:
I guess you didn't use `just generate-readmes` to generate this file? I don't think it's checked by CI at the moment, but could lead to problems in the future.

Reply:
Nah, I'm just using windows ^^. Is there something wrong here?

Reply:
If you have allowed edits by maintainers I can run the script if you like.

Reply:
Honestly, I'd prefer not generating the alt texts. Allow edits is on by default.

Reply:
OK. Done.

Reply:
Thank you